RELEASE.md

```diff
@@ -0,0 +1,10 @@
+# Release Process
+
+The process of creating a release is described in this document. Replace `X.Y.Z` with the version to be released.
+
```

Comment:
I'd say a couple of words that X.Y.Z should be digits, and we can use labels such as X.Y.Z-alpha
 https://semver.org/

Reply:
Done. Thanks